.gitignore

```diff
@@ -194,8 +194,13 @@ src/lib/Libpbs/ecl_resc_def_all.c
 src/lib/Libpbs/ecl_resv_attr_def.c
 src/lib/Libpbs/ecl_sched_attr_def.c
 src/lib/Libpbs/ecl_svr_attr_def.c
+src/include/stamp-h1
+src/lib/Libpython/pbs_ifl.i
+src/lib/Libpython/pbs_ifl.py
+src/lib/Libpython/pbs_ifl_wrap.c
 
 #Generated source files - Windows
+win_configure/include/pbs_version_win.h
```

Comment:
You are adding pbs_version_win.h to .gitignore; however this file is actually still part of the repository. If this is indeed an autogenerated file, then it should be removed from the repo